nss-day-cohort-29/reactive-nutshell-starrywick
# Added messages component. Messages appear sorted with timestamp in ascending order. Still need to associate username with message

# Description

This branch adds messages to DOM with user id, content, and is sorted by timestamp in ascending order


## Type of change

- [X ] New feature (non-breaking change which adds functionality)



# Testing Instructions for Change Made

git fetch --all
git checkout branch db-message-start-up
run json server on port 5002
npm start in reactive-nutshell
click on messages button in NavBar 
A list of messages should appear
try adding and deleting messages of your choice
timestamp will appear on any newly added messages.
check your database.json to see that a timestamp occurs on the new messages as a key in messages

here is a sample database
  "messages": [
    {
      "id": 2,
      "userId": "1",
      "content": "hhh",
      "timeStamp": "1/31/2019, 11:18:48 AM"
    },
    {
      "id": 3,
      "userId": "2",
      "content": "qqqqq",
      "timeStamp": "1/31/2019, 11:26:18 AM"
    }
  ],


# Checklist:

- [X ] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [X ] My changes generate no new warnings
- [X ] I have added test instructions that prove my fix is effective or that my feature works
